DotNet/Curly/Stdlib.cs

```diff
@@ -304,6 +304,18 @@ public static string EnumValues([FromParam(0)] string enumName, [FromParam(1)] s
         /// <returns></returns>
         [DslMethod("equals")]
         public static string Equals([FromParam(0)] string first, [FromParam(1)] string second)
+        {
+            return string.Equals(first.ToLower(), second.ToLower()).ToString();
```

Comment:
First of all - lets do opposite. Make equals-ignore-case method. And implementation will be better with ordinals
```suggestion
    return string.Equals(first, second,StringComparison, OrdinalIgnoreCase).ToString();
```

Reply:
All curly methods are case insensitive except this one. I think that it's a bug. So we need to fix it and add another method which will be case sensitive.

Reply:
Yes, you are right.